ROADMAP.md

```diff
@@ -0,0 +1,24 @@
+# Roadmap
+The roadmap is for information only. The listed items may be added, removed, or changed due to priority change, dependency readiness and customer demands, etc.
```

Comment:
This roadmap for Teams Toolkit is provided for informational purposes. The roadmap may change as design and development progresses. For feedback on the roadmap and to add items for consideration, log a Github issue.

Reply:
What do you think about logging a github issue? Is that our process?

Reply:
this is a place holder file, and please ignore the current content in this ROADMAP.md, John has been working on the final version based on a Pierce's changed structure, for Monday US time. so will replace content with the final version.

For user to open github issue. it is in the last section of SUPPORT.md